e2e/package.json

```diff
@@ -9,10 +9,10 @@
     "test": "cross-env jest --maxWorkers=5",
     "test:debugTest": "cross-env PUPPETEER_HEADLESS=false DEBUG=true node --inspect-brk node_modules/.bin/jest",
     "test:debug": "cross-env PUPPETEER_HEADLESS=false DEBUG=true jest --detectOpenHandles",
-    "test-local": "cross-env WORKBENCH_ENV=local DEBUG=true jest --maxWorkers=3",
-    "test-local:debug": "cross-env WORKBENCH_ENV=local PUPPETEER_HEADLESS=false DEBUG=true jest --maxWorkers=3",
+    "test-local": "cross-env WORKBENCH_ENV=local DEBUG=true jest --detectOpenHandles",
```

Comment:
`--maxWorkers=3` will cause test to fail when only one test is selected to run.

Reply:
This is surprising.  Where have you seen this?

Also: why change some of these but not others, and what does adding `detectOpenHandles` do?

Reply:
The error is `Protocol error (DOM.describeNode): Target closed.` when running only one test on localhost like `yarn test-local tests/home/home-page.spec.ts`.
`--detectOpenHandles` detects one code issue: when `asyn/promise` code didn't use `await`.

Reply:
I tried this out, and I was able to run a single test with maxWorkers=3.  Let's keep that flag.
```
thibault@wm98d-e39 ~/src/aou/e2e (joel/create-auth-domain) $ yarn test-local tests/home/home-page.spec.ts                                                                                                                                           yarn run v1.21.1                                                                                                                                                                                                                                    $ cross-env WORKBENCH_ENV=local DEBUG=true jest --maxWorkers=3 tests/home/home-page.spec.ts                                                                                                                                                          PASS  tests/home/home-page.spec.ts (35.393 s)                                                                                                                                                                                                        Home page ui tests                                                                                                                                                                                                                                    ✓ Check visibility of Workspace cards (16831 ms)                                                                                                                                                                                                    ✓ Check Create New Workspace link on Home page (15654 ms)                                                                                                                                                                                                                                                                                                                                                                                                                                           Test Suites: 1 passed, 1 total                                                                                                                                                                                                                      Tests:       2 passed, 2 total                                                                                                                                                                                                                      Snapshots:   0 total                                                                                                                                                                                                                                Time:        36.542 s                                                                                                                                                                                                                               Ran all test suites matching /tests\/home\/home-page.spec.ts/i.                                                                                                                                                                                     ✨  Done in 39.23s.
```

Reply:
`detectOpenHandles` looks useful.  Thanks 👍

Reply:
Reverted package.json back to original, saving `detectOpenHandles` change for another day.